guides/products_and_variants.md

```diff
@@ -0,0 +1,135 @@
+
+* Small, Red
+* Small, Green
+* Small, Blue
+* Medium, Red
+* Medium, Green
+* Medium, Blue
+* Large, Red
+* Large, Green
+* Large, Blue
+
+## Images
+
+Images link to a product through its master variant. The sub-variants for the product may also have their own unique images to differentiate them in the frontend.
+
+Solidus automatically handles creation and storage of several size versions of each image (via the Paperclip gem, documentation [here](https://github.com/thoughtbot/paperclip)). The default sizes can be changed by placing something like the below config in the paperclip initializer (/config/initializers/paperclip.rb - note that the below example sizes are the default). 
+
+```ruby
+Spree::Image.attachment_definitions[:attachment][:styles] = {
+  mini: '48x48>',
+  small: '100x100>',
+  product: '240x240>',
+  large: '600x600>',
+}
+```
+
+You *must* regenerate the paperclip thumbnails by running this command:
+
+```bash
+$ bundle exec rake paperclip:refresh:thumbnails CLASS=Spree::Image
+```
+
+If you want to change the image that is displayed when a product has no image, simply create new versions of the files within [Solidus' app/assets/images/noimage directory](https://github.com/solidusio/solidus/tree/master/core/app/assets/images/noimage). These image names must match the keys defined in the paperclip config.
+
+## Product Properties
+
+Product properties track individual attributes for a product which don't apply to all products. These are typically additional information about the item. For instance, a T-Shirt may have properties representing information about the kind of material used, as well as the type of fit the shirt is.
+
+A `Property` should not be confused with an [`OptionType`](#option_type), which is used when defining [Variants](#variants) for a product.
+
+You can retrieve the value for a property on a `Product` object by calling the `property` method on it and passing through that property's name:
+
+```bash
+$ product.property("material")
+=> "100% Cotton"
+```
+
+You can set a property on a product by calling the `set_property` method:
+
+```ruby
+product.set_property("material", "100% cotton")
+```
+
+If this property doesn't already exist, a new `Property` instance with this name will be created.
+
+## Multi-Currency Support
+
+`Price` objects track a price for a particular currency and variant combination. For instance, a [Variant](#variants) may be available for $15 (15 USD) and €7 (7 Euro).
+
+This presence or lack of a price for a variant in a particular currency will determine if that variant is visible in the frontend. If no variants of a product have a particular price value for the site's current currency, that product will not be visible in the frontend.
+
+You may see what price a product would be in the current currency (`Spree::Config[:currency]`) by calling the `price` method on that instance:
+
+```bash
```

Comment:
Good catch - missed this when porting from the Spree version -- irb/console is definitely ruby.  Fixing.